ifStatement.js

```diff
@@ -59,7 +88,16 @@ Steps:
     - If the user is younger than 18, the program should calculate how many years are left until they turn 18 and print the message "You will be eligible to vote in X years", where X is the number of years left.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
-
+let Age=prompt("ENTRE AGE ");
```

Comment:
We don't capitalize the first letter of the first word when naming a variable => age not Age